refarch-apigateway/apigateway/src/main/resources/application-test.yml

```diff
@@ -0,0 +1,63 @@
+wiremock:
+  server:
+    port: 0
+    https-port: -1
+
+spring:
+  
+  # muenchen.de realm is used by default so that the SpringContext can be initalized upon running the tests. Change to your own dev realm if available
+  realm: muenchen.de
```

Comment:
Having dependency on muenchen.de for running local tests is not acceptable. We need a different solution here (e.g. a local keycloak instance if needed)

Reply:
maybe we can use [Testcontainers ](https://www.baeldung.com/spring-boot-keycloak-integration-testing). Issues created #39